src/core/frames/frame_controller.ts

```diff
@@ -314,6 +314,12 @@ function readScrollLogicalPosition(value: string | null, defaultValue: ScrollLog
 function fragmentFromHTML(html?: string) {
   if (html) {
     const foreignDocument = document.implementation.createHTMLDocument()
```

Comment:
@sstephenson Would you be able to comment on why it was needed to use a "foreign document" here? I'm asking because it would be possible to use `createContextualFragment()` on the current document and that also seems to work.

In short, this combination of a foreign document with `createContextualFragment()` apparently prevents Firefox to load SVG sprite icons (see the original issue). It would be interesting to understand the rationale behind using it.